test/acceptance/features/steps/util.py

```diff
@@ -1,5 +1,10 @@
 import os
-
+from string import Template
 
 def scenario_id(context):
     return f"{os.path.basename(os.path.splitext(context.scenario.filename)[0]).lower()}-{context.scenario.line}"
+
+def get_scenario_id(context, text=None):
```

Comment:
the function does not return `scenario_id`, it take as input some sort of template and process it, replacing variables like `$scenario_id` with a concrete value.  Can you come up with some function name that reflect that?

Reply:
How does `substitute_scenario_id` sound?  Alternatively, `replace_scenario_id`?

Reply:
+1 for `substitute_scenario_id`